### PR TITLE
Minor update to readme

### PR DESCRIPTION
**Pull request by seungpark**
Added a minor bit of components used in search server

[Trivial follow up task](https://jira.mongodb.org/browse/DOP-3771) to separate the Marian server code.

Pls comment if anything else can be cleared up!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,17 @@
 
 REST server that indexes search documents within an Atlas cluster and handles user search requests, deployed via Kanopy
 
+## Components
+
+### Marian Server
+REST server that exposes endpoints to run queries against A DB.
+
+### Atlas Admin Manager
+Module to invoke Atlas Admin commands via the Atlas Admin API.
```
Comment (rayangler) [inline README.md:11]: (Non-blocking suggestion) Would it be a good idea to include a sentence to provide an overview/more context on what the purpose of these commands are for? Something like `"The module syncs the Search Index to ensure content taxonomy is incorporated in faceted search data."` ?

File: README.md [hunk 2]
```diff
@@ -2,6 +2,17 @@
 
 REST server that indexes search documents within an Atlas cluster and handles user search requests, deployed via Kanopy
 
+## Components
+
+### Marian Server
+REST server that exposes endpoints to run queries against A DB.
```
Comment (rayangler) [inline README.md:8]: (Super duper nit) Should we lowercase "A DB" to "a DB"? Or does the "A" stand for "Atlas" in this case?

**Review by rayangler — COMMENTED** [review]
Looks good! Just wanted to learn more about the Atlas Admin Manager and was wondering if an extra sentence might help with context. If unnecessary, I'm okay with dropping it!

File: README.md (hunk 2 above)
Comment (seungpark) [reply README.md]: Atlas DB is correct :) thanks for the catch!

File: README.md (hunk 1 above)
Comment (seungpark) [reply README.md]: good suggestion! i didn't want to explain specific details of projects so i added the admin API, as well as the Search Index. lmk if this works!

**Review by rayangler — APPROVED** [review]

**Pull request merged**